addons/hr_holidays/models/hr_holidays_status.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Copyright (c) 2005-2006 Axelor SARL. (http://www.axelor.com)
+
+from odoo import api, fields, models
+
+
+class HrHolidaysStatus(models.Model):
+    _name = "hr.holidays.status"
+    _description = "Leave Type"
+
+    name = fields.Char(string='Leave Type', required=True, translate=True)
+    categ_id = fields.Many2one('calendar.event.type', 'Meeting Type',
+        help='Once a leave is validated, Odoo will create a corresponding meeting of this type in the calendar.')
+    color_name = fields.Selection([('red', 'Red'),('blue','Blue'), ('lightgreen', 'Light Green'), ('lightblue','Light Blue'), ('lightyellow', 'Light Yellow'), ('magenta', 'Magenta'),('lightcyan', 'Light Cyan'),('black', 'Black'),('lightpink', 'Light Pink'),('brown', 'Brown'),('violet', 'Violet'),('lightcoral', 'Light Coral'),('lightsalmon', 'Light Salmon'),('lavender', 'Lavender'),('wheat', 'Wheat'),('ivory', 'Ivory')],'Color in Report', required=True, help='This color will be used in the leaves summary located in Reporting\Leaves by Department.', default='red')
+    limit = fields.Boolean('Allow to Override Limit', help='If you select this check box, the system allows the employees to take more leaves than the available ones for this type and will not take them into account for the "Remaining Legal Leaves" defined on the employee form.')
+    active = fields.Boolean('Active', help="If the active field is set to false, it will allow you to hide the leave type without removing it.", default=True)
+    max_leaves = fields.Float(compute='_compute_user_left_days', string='Maximum Allowed', help='This value is given by the sum of all holidays requests with a positive value.')
+    leaves_taken = fields.Float(compute='_compute_user_left_days', string='Leaves Already Taken', help='This value is given by the sum of all holidays requests with a negative value.')
+    remaining_leaves = fields.Float(compute='_compute_user_left_days', string='Remaining Leaves', help='Maximum Leaves Allowed - Leaves Already Taken')
+    virtual_remaining_leaves = fields.Float(compute='_compute_user_left_days', string='Virtual Remaining Leaves', help='Maximum Leaves Allowed - Leaves Already Taken - Leaves Waiting Approval')
+    double_validation = fields.Boolean('Apply Double Validation', help="When selected, the Allocation/Leave Requests for this type require a second validation to be approved.")
+    company_id = fields.Many2one('res.company', 'Company')
+
+    def get_days(self, employee_id):
+        result = dict.fromkeys(self.ids, dict(max_leaves=0, leaves_taken=0, remaining_leaves=0, virtual_remaining_leaves=0))
```

Comment:
This is going to create a single subdict mapped to every keys, so the code is completely broken if there's more than one `holiday` matched.

Also use dict literals.